src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -84,6 +85,13 @@ public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolut
             CurrentConfiguration = configurationProvider.GetConfiguration();
 
             SetBoundSolutionUIContext();
+
+            gitEventsMonitor.HeadChanged += GitEventsMonitor_HeadChanged;
+        }
+
+        private void GitEventsMonitor_HeadChanged(object sender, EventArgs e)
+        {
+            SolutionBindingUpdated?.Invoke(this, EventArgs.Empty);
```

Comment:
This means the tracker will raise the event even for standalone solutions. Is this what we expect?

Reply:
It handles all solution open/closed events from the lower-level `IActiveSolutionTracker`. It will raise the `SolutionBindingChanged` event for standalone solutions/when a solution is closed.

I don't think it will raise it will raise the `SolutionBindingUpdated` event except for up bound solutions, but I could be wrong.